plugin/src/main/java/org/opennms/velocloud/rest/api/VelocloudRestService.java

```diff
@@ -46,4 +51,24 @@ public interface VelocloudRestService {
     @Path("/partner/{alias}/customers")
     @Produces(value = {MediaType.APPLICATION_JSON})
     List<EnterpriseDTO> getCustomersForMspPartner(@PathParam("alias") String alias) throws VelocloudApiException;
+
+    @GET
+    @Path("/connections/list")
+    @Produces(value = {MediaType.APPLICATION_JSON})
+    List<ConnectionListElementDTO> getConnectionList() throws VelocloudApiException;
+
+    @POST
+    @Path("/connections/{alias}/add")
+    @Consumes({MediaType.APPLICATION_JSON})
+    Response addConnection(@PathParam("alias") String alias, @QueryParam("dryrun") boolean dryRun, ConnectionDTO connectionDTO) throws VelocloudApiException;
+
+    @POST
+    @Path("/connections/{alias}/edit")
+    @Consumes({MediaType.APPLICATION_JSON})
+    Response editConnection(@PathParam("alias") String alias, ConnectionDTO connection) throws VelocloudApiException;
+
+    @PUT
+    @Path("/connections/{alias}/validate")
```

Comment:
```suggestion
    @GET
    @Path("/connections/{alias}")
```

We transfer the validation state here. Therefore there should be a DTO having the alias, the url and the validation state.